### PR TITLE
Update instrumentation.md

### PR DESCRIPTION
**Pull request by RichiH**
Try to trigger rebuild; might fix https://github.com/prometheus/docs/issues/1045

### CODE REVIEW

**Comment by RichiH** [comment]
Changed the first thing I could find to nitpick.

**Comment by RichiH** [comment]
Heh, DCO is an issue with in-web changes.

File: content/docs/practices/instrumentation.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This page provides an opinionated set of guidelines for instrumenting your code.
 ## How to instrument
 
 The short answer is to instrument everything. Every library, subsystem and
-service should have at least a few metrics to give you a rough idea of how it is
+service should expose at least a few metrics to give you a rough idea of how it is
```
Comment (brian-brazil) [inline content/docs/practices/instrumentation.md]: Instrumentation and exposition are separate concerns, libraries should not be exposing metrics. They should register them in a registry.

**Comment by RichiH** [comment]
Erm, wrong PR. Disregard my confusion.

**Comment by beorn7** [comment]
DCO is fine as per human inspection.

**Pull request merged**